shared/src/mobileMain/kotlin/org/jetbrains/kotlinconf/ui/components/SearchField.kt

```diff
@@ -0,0 +1,64 @@
+package org.jetbrains.kotlinconf.ui.components
+
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material.Button
+import androidx.compose.material.ButtonDefaults
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Text
+import androidx.compose.material.TextField
+import androidx.compose.material.TextFieldDefaults
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import org.jetbrains.kotlinconf.theme.grey5Black
+import org.jetbrains.kotlinconf.theme.greyWhite
+import org.jetbrains.kotlinconf.theme.orange
+
+@Composable
+fun SearchField(text: String, onTextChange: (String) -> Unit) {
+    Box(
+        Modifier
+            .fillMaxWidth()
+            .padding(0.dp)
+    ) {
+        Row(Modifier.fillMaxWidth()) {
+            TextField(
+                value = text,
+                onValueChange = onTextChange,
+                modifier = Modifier.fillMaxWidth(),
+                textStyle = MaterialTheme.typography.body2.copy(
+                    color = MaterialTheme.colors.greyWhite
+                ),
+                maxLines = 1,
+                colors = TextFieldDefaults.outlinedTextFieldColors(
+                    backgroundColor = MaterialTheme.colors.grey5Black,
+                    cursorColor = orange,
+                    textColor = MaterialTheme.colors.greyWhite,
+                    focusedBorderColor = MaterialTheme.colors.grey5Black
+                ),
+            )
+        }
+        Row(Modifier.fillMaxWidth()) {
+            Spacer(Modifier.weight(1f))
```

Comment:
to arrange an element on the right you may use an alignment instead of the spacer here

Reply:
thanks, done